web/.eslintrc.json

```diff
@@ -22,6 +22,7 @@
     "react/prefer-stateless-function": 0,
     "react/no-did-mount-set-state": 0,
     "react/jsx-one-expression-per-line": 0,
+    "react/button-has-type": 1,
```

Comment:
Due to https://github.com/yannickcr/eslint-plugin-react/issues/1846 and https://github.com/yannickcr/eslint-plugin-react/issues/1555